udata/commands/tests/test_fixtures.py

```diff
@@ -0,0 +1,26 @@
+import json
+from tempfile import NamedTemporaryFile
+
+import pytest
+
+from udata import models
+from udata.tests import DBTestMixin, TestCase
+
+
+class FixturesTest(DBTestMixin):
+    @pytest.mark.datagouvfr
+    @pytest.mark.options(FIXTURE_DATASET_SLUGS=["nombre-de-personnes-rickrollees-sur-data-gouv-fr"])
+    def test_generate_fixtures_file_and_generate_fixtures(self, cli):
+        with NamedTemporaryFile(mode="w+", delete=True) as fixtures_fd:
+            # Get the fixtures from http://data.gouv.fr.
+            result = cli("generate-fixtures-file", "http://data.gouv.fr", fixtures_fd.name)
```

Comment:
I'm not exactly clear what you would like to see to move this PR forward. I agree that testing against the prod env isn't ideal. Also note that the previous test wasn't testing the fixtures generation at all.

So maybe, i could:

1. remove the part of the test that generates the fixtures (back to the original code which wasn't testing it)
2. modify the part of the test that imports those fixtures to read from a local `results_test.json` file (generated from prod for now, and committed to the repository): the previous code was failing, the fixtures from the test files won't load in the db as is
3. create a future PR that generates this `results_test.json` file from the current env, and makes sure the newly created `results_test.json` file is no different than the local one

This way, once the new PR is landed, we'd have two decoupled tests, and the fixtures would be generated from a local env.

What do you think?

Reply:
> For 2., I would use the DEFAULT_FIXTURE_FILE for now.
Yes indeed, that's simpler, and it would make sure the "official" fixtures file is up to date and still working as expected, good idea 👍 

>For 3., I'm not sure about comparing the generated file with a snapshot one, we just want to make sure that importing it succeed with the correct content being imported. It would prevent making the test fail on any model modification as long as fixtures get generated and loaded without issues.
I see, if I understand that correctly, we would have two tests, one that is making sure that the `DEFAULT_FIXTURE_FILE` is still compatible, and a second one that would first generate some data in the db, then `generate_fixtures_file` and then `import_fixtures` with it (to make sure the data we extract can be re-imported).

Is that what you had in mind?

Reply:
Yes exactly! :pray: Seems to cover any modifications that would break the fixtures generation or importation :)

Reply:
Ok, so I created a simpler, more focused PR #3105 to fix the non running tests for 1.
And a PR (draft) #3106 to address the rest of this comment (2. and 3.)